addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -11666,6 +11666,40 @@ QUnit.module("Views", (hooks) => {
         assert.verifySteps([], "should not save because we do not change anything");
     });
 
+    QUnit.test("Auto save: save on closing tab/browser (not dirty) with text field", async function (assert) {
+
+        serverData.models.partner.fields.bloup = {
+            string: "Bloup",
+            type: "text",
+            default: false
```

Comment:
typical picky std remarks (for a better git history if someone one day want to add a line after)
```suggestion
            default: false,
```